src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -142,13 +142,13 @@ const ProjectInstallPlatform = React.createClass({
             <div dangerouslySetInnerHTML={{__html: this.state.html}}/>
           )}
 
-          {queryParams.onboarding ?
+          {'signup' in queryParams ?
```

Comment:
Returns empty string, which evals to false. I could do `queryParams.signup != null` instead, if preferred.